tests/codegen/params_count.rs

```diff
@@ -0,0 +1,24 @@
+//! Test that the generated code has the right number of parameters
+
+#![feature(no_core)]
+#![no_core]
+#![no_main]
+
+extern crate mini_core;
+
+// CHECK-LABEL: foo
+// CHECK-LABEL: main
```

Comment:
I think `CHECK-LABEL` should only be used when there are other `CHECK`s associated with it, otherwise a normal `CHECK` is fine

```suggestion
// CHECK: int32_t foo
// CHECK: int32_t main
```

Reply:
`CHECK-LABEL` is required here to skip the function declaration. Otherwise, the `CHECK-LABEL` at line 13 would match the function declaration instead of the function definition, resulting in an error.